anomaly_detector/adapters/som_model_adapter.py

```diff
@@ -175,6 +174,8 @@ def prediction_builder(self, data, false_positives, json_logs, threshold):
 
     def process_anomaly_score(self, data):
         """Generate scores from some. To be used for inference."""
+        if data.empty is True:
+            raise EmptyDataSetNotAllowed()
         v = self.w2v_model.one_vector(data)
```

Comment:
When a pandas dataframe is passed into this function it causes a crash. We need to validate before we run one_vector function.